src/main/java/org/venice/beachfront/bfapi/controllers/HealthCheckController.java

```diff
@@ -70,12 +73,13 @@
 	@ApiResponses(value = {
 			@ApiResponse(code = 200, message = "Health check information, including server uptime and job counts", response = HashMap.class),
 			@ApiResponse(code = 500, message = "Unexpected internal server error", response = JsonNode.class) })
-	public Map<String, String> healthCheck() throws UserException {
-		Map<String, String> healthCheckData = new HashMap<>();
+	public Map<String, Object> healthCheck() throws UserException {
+		Map<String, Object> healthCheckData = new HashMap<>();
 		// Show uptime
 		healthCheckData.put("uptime", Double.toString(uptimeService.getUptimeSeconds()));
 		healthCheckData.put("geoserver-upstream", geoserverEnvironment.getGeoServerBaseUrl());
 		healthCheckData.put("geoserver", String.format("https://bf-api.%s/geoserver", domain));
+		healthCheckData.put("enabled-platforms", sceneService.getEnabledPlatforms());
```

Comment:
I like the implementation, but isn't this literally a "data source"? "platform" seems overly ambiguous.

Reply:
"Platform" is the terminology used elsewhere in BF-API for these things, so I was going for consistency.